Comparing source compatibility of commons-configuration2-2.11.0.jar against commons-configuration2-2.10.1.jar
**** MODIFIED CLASS: PUBLIC ABSTRACT org.apache.commons.configuration2.AbstractConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean contains(java.util.Iterator<java.lang.String>, java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) boolean containsValue(java.lang.Object)
	+++* NEW METHOD: PROTECTED(+) ABSTRACT(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.commons.configuration2.AbstractHierarchicalConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	GENERIC TEMPLATES: === T:java.lang.Object
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.BaseConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.CompositeConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.DatabaseConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC javax.sql.DataSource getDatasource()
		+++  NEW ANNOTATION: java.lang.Deprecated
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.DataConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.DynamicCombinedConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.commons.configuration2.ImmutableConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) boolean containsValue(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.JNDIConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.MapConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.PatternSubtreeConfigurationWrapper  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)
***  MODIFIED CLASS: PUBLIC org.apache.commons.configuration2.SubsetConfiguration  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PROTECTED(+) boolean containsValueInternal(java.lang.Object)

Semantic versioning suggestion: 0.1.0